apps/server/src/shared/domain/entity/import-user.entity.ts

```diff
@@ -72,8 +72,8 @@ export class ImportUser extends BaseEntityWithTimestamps implements IEntityWithS
 		return null;
 	}
 
-	@Property()
-	ldapId: string;
+	@Property({ fieldName: 'ldapId' })
+	externalId: string;
```

Comment:
Maybe we should create a refactoring ticket for that. I think at the moment it is fine, but in the future a migration script for renaming the column and remove the mapping here would be fine